.bumpversion.cfg

```diff
@@ -1,7 +1,7 @@
 [bumpversion]
 commit = False
 tag = True
-current_version = 1.1.174
+current_version = 1.1.175
```

Comment:
Why do you change it manually ?

Reply:
The previous build failed on the Pypi upload. I didn't notice the fail.